### PR TITLE
Feature : Update readme with proper & valid content [fixes #6]

### LINKED ISSUE #6: Update readme with proper & valid content.

### PR DESCRIPTION
**Pull request by najeebkp**
### Feature : Update readme with proper & valid content [fixes #6]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,54 @@
...
+```markdown
+git clone https://github.com/your-username/Hacktoberfest2022.git
+```
+
+3. change the present working directory
+
+```markdown
+cd Hacktoberfest2022
+```
+
```
Comment (nirajpdn) [inline README.md:19]: Where is 4 ? 
You can add:
- Create new branch
- ```git checkout -b <branch-name>```

Comment (najeebkp) [reply README.md:24]: Done

**Review by nirajpdn — CHANGES_REQUESTED** [review]
Some changes needed.

**Review by nirajpdn — APPROVED** [review]
LGTM !!

**Comment by nirajpdn** [comment]
@najeebkp , Thanks for your contribution.

**Pull request merged**